apps/labs/posts/what-do-we-need-to-write.md

```diff
@@ -0,0 +1,135 @@
+---
+title: 'Why is writing blog posts hard?'
+published: April 10, 2022
+author: tania-allard, melissa-mendonca, isabela-presedo-floyd, rohit-goswami, tony-fast, dillon-roach, dharhas-pothina, eric-kelly
```

Comment:
@noatamir you're already aware of this, I think, but multiple authors seems not supported at the moment. This line causes the build to fail. I'm not sure what we should do. cc @trallard